main.py

```diff
@@ -34,7 +34,7 @@ def parse_args():
     
     parser.add_argument(
         "--model",
-        required=True,
+        default="",
```

Comment:
I suspect this will crash with an uninformative message if the user doesn't supply a valid model name

Reply:
Actually for the evaluation only model (no text generation) you don’t need a valid model name, this argument is just saved with the results for documenting the experiment. 
But I'll improve the help message and put a valid model name for illustration